### PR TITLE
fix(v4): remove is-svg dependency

### LINKED ISSUE #1030: CVE-2021-28092 in `cssnano` due to an old version of `is-svg`
**Describe the bug**
[CVE-2021-28092](https://nvd.nist.gov/vuln/detail/CVE-2021-28092)

> The is-svg package 2.1.0 through 4.2.1 for Node.js uses a regular expression that is vulnerable to Regular Expression Denial of Service (ReDoS). If an attacker provides a malicious string, is-svg will get stuck processing the input for a very long time.

Looking at the dependency tree of css nano shows that postcss-svgo depends on is-svg@3.0.0

```
└─┬ react-scripts@4.0.3
  └─┬ optimize-css-assets-webpack-plugin@5.0.4
    └─┬ cssnano@4.1.10
      └─┬ cssnano-preset-default@4.0.7
        └─┬ postcss-svgo@4.0.2
          └── is-svg@3.0.0
```

### PR DESCRIPTION
**Pull request by AviVahl**
backport of #1034 to the `v4` branch.

closes #1030

### CODE REVIEW

**Comment by AviVahl** [comment]
The `yarn.lock` changes were generated by executing latest yarn@1 on latest node@10.
Tests pass locally.

**Comment by AviVahl** [comment]
Tested locally on Node v6.17.1 as well. Tests pass.

File: packages/postcss-svgo/src/index.js [hunk 1]
```diff
@@ -75,6 +70,9 @@ function minifyPromise (decl, getSvgo, opts) {
                     });
                 })
                 .catch(error => {
+                    if (String(error).startsWith('Error in parsing SVG')) {
+                        return;
+                    }
                     throw new Error(`${PLUGIN}: ${error}`);
```
Comment (alexander-akait) [inline packages/postcss-svgo/src/index.js:76]: Let's do warning here too, as in original PR

Comment (AviVahl) [reply packages/postcss-svgo/src/index.js]: sorry, didn't notice the PR had another commit. rebased changes.

**Review by alexander-akait — APPROVED** [review]
/cc @ludofischer

File: packages/postcss-svgo/src/__tests__/index.js [hunk 2]
```diff
@@ -152,6 +152,13 @@ test(
     'h1{background-image:url("data:image/svg;charset=US-ASCII,<?xml version="1.0" encoding="utf-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve"><circle cx="50" cy="50" r="40" fill="yellow" /><!--test comment--></svg>")}',
 );
 
+test('should warn on SVG containing unclosed tags', async (t) => {
+    const css =
+      'h1{background:url(data:image/svg+xml;charset=utf-8,<svg>style type="text/css"><![CDATA[ svg { fill: red; } ]]></style></svg>)}';
+    const result = await postcss(plugin()).process(css, {from: undefined});
+    t.deepEqual(result.messages.length, 1);
```
Comment (ludofischer) [inline packages/postcss-svgo/src/__tests__/index.js:159]: Good that this test is here, although `deepEqual()` is strange (there's nothing 'deep' about `1`). But it does not hurt wither I guess.

**Review by ludofischer — APPROVED** [review]
LGTM, I am not familiar with the PostCSS 7 API but since the tests pass I think there can't be a mistake at that level.

File: packages/postcss-svgo/src/__tests__/index.js (hunk 2 above)
Comment (AviVahl) [reply packages/postcss-svgo/src/__tests__/index.js]: fixed to `t.is`

**Pull request merged**